README.adoc

```diff
@@ -0,0 +1,85 @@
+:imagesdir: ./screenshots
+
+= Visualize Sync Metrics Data on Openshift
+
+== Overview
+
+This guide will go through the steps to help you setup Influxdb and Grafana on Openshift to receive metrics data from the RHMAP sync framework, and view them in a Grafana dashboard. At the end, you should have a dashboard look like this:
```

Comment:
In other documents it looks like we're capitalising the `Sync Framework` part of the name so we should probably do the same here to be consistent.

Quite minor but the branding for InfluxDB seems to use `InfluxDB` when referring to itself so maybe we should use that too. We have some `influxdb`'s and `influx db`'s currently.